### PR TITLE
Correct coding style link

### PR DESCRIPTION
**Pull request by madstk1**
The link to the coding style document in the README was invalid. This might help new developers, myself included, during Hacktoberfest and/or otherwise.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ We have other projects tightly related to ArkScript, which aren't necessarily C+
 
 ### Contributing to the ArkScript standard library
 
-See [Coding guidelines](https://github.com/ArkScript-lang/Ark/wiki/Coding-guidelines#coding-in-ark) if you want to write ArkScript for the library (see folder `lib/std/`).
+See [Coding guidelines](https://github.com/ArkScript-lang/Ark/blob/dev/CONTRIBUTING.md#c-coding-guidelines) if you want to write ArkScript for the library (see folder `lib/std/`).
```
Comment (SuperFola) [inline README.md:127]: This is actually the link to the C++ contributing guidelines, the ArkScript ones are below (on the page you linked)!

May I suggest renaming the section (Contributing to the ArkScript standard library) to `Coding guidelines for contributing` and linking the cpp contribution guidelines + the arkscript ones?

Something like a link if you want to contribute to cpp code, and another if you want to contribute to arkscript code

Comment (madstk1) [reply README.md]: I've commited an update.

File: README.md [hunk 2]
```diff
@@ -122,9 +122,11 @@ We have other projects tightly related to ArkScript, which aren't necessarily C+
 
 [Full list here](https://github.com/ArkScript-lang/Ark/graphs/contributors).
 
-### Contributing to the ArkScript standard library
+### Coding guidelines for contributing
 
-See [Coding guidelines](https://github.com/ArkScript-lang/Ark/blob/dev/CONTRIBUTING.md#c-coding-guidelines) if you want to write ArkScript for the library (see folder `lib/std/`).
+See [C++ Coding guidelines](https://github.com/ArkScript-lang/Ark/blob/dev/CONTRIBUTING.md#c-coding-guidelines) if you want to write ArkScript for the library (see folder `lib/std/`).
```
Comment (SuperFola) [inline README.md:127]: The link is for C++, thus "if you want to write ArkScript for the library (see folder `lib/std/`)." should be more like "if you want to contribute to ArkScript compiler / runtime".

**Review by SuperFola — APPROVED** [review]
Looks good to me!

**Comment by SuperFola** [comment]
I'll squash the modifications into a single commit since it's the same thing, is that okay?

**Comment by madstk1** [comment]
Yes, of course. Go ahead.

**Pull request merged**